src/player/player.rs

```diff
@@ -1,8 +1,9 @@
 use crate::class::class::*;
 
+#[derive(Debug)]
 pub struct Player {
     pub pid: usize,
-    pub pos: (usize, usize),
+    pub pos: (isize, isize),
```

Comment:
should be a usize probably but i didnt want to implement conversion functions for math things so i just as isize for now...
will do later